### DISCUSSION TITLE
Add TPCDS connector in Velox

### DISCUSSION THREAD

**Discussion by majetideepak** [facebookincubator/velox#8994] [General]
We should implement a TPCDS connector in Velox to generate data and run benchmarks against it.
The implementation stages would be similar to the TPCH connector.
TPCDS dsdgen can also generate delta files. This can be used to test the Lakehouse formats such as Iceberg.

**Comment by aditi-pandit** [comment]
Updating status on this. @pdabre12 from IBM has some implementation that they are working to Open source in the future.

**Comment by aditi-pandit** [comment]
We spoke internally among IBM engineers. Since this work is mostly for writing TPC-DS microbenchmarks and e2e integration tests (in presto-native-execution(tests) module), we prefer to have this in Presto repo instead of Velox right now. We could move velox parts to the velox repo at a future time.

Pratik will post a design doc in #9523 for the changes.

@majetideepak @tdcmeehan @yingsu00 @pdabre12
